test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.test.js

```diff
@@ -1739,6 +1739,36 @@ describe('#compileMethods()', () => {
         .ApiGatewayMethodUsersCreatePost.Properties
     ).to.not.have.key('OperationName');
   });
+
+  it('should depends on permission resource', async () => {
+    const {
+      awsNaming,
+      cfTemplate: { Resources: cfResources },
+    } = await runServerless({
+      command: 'package',
+      fixture: 'apiGateway',
+      configExt: {
```

Comment:
Do we need that configuration extension? The `apiGateway` fixture already configures some events that I believe we can use in this test